docs/cn/MatrixOne/TempTable/tempTable.md

```diff
@@ -0,0 +1,28 @@
+## **临时表设计**
+
+临时表的整体设计架构和语义请参考[TempTable skeleton design](https://github.com/matrixorigin/docs/tree/main/pipeline/temporary_table)
```

Comment:
maybe md file move to https://github.com/matrixorigin/docs is better
@yingfeng

Reply:
> 

all docs under matrixone/docs/cn/en have been removed in pr #6347 and will be transferred to a dedicate store decided by @dengn